src/components/ConferenceNewPage/ConferenceNewPage.tsx

```diff
@@ -402,7 +418,7 @@ const ConferenceNewPage: React.FC = () => {
                   value={twitter}
                   onChange={handleFieldChange}
                 />
-                {errorFor('twitter', 'Twitter handle is required.')}
+                {errorFor('twitter', 'Twitter handle should be valid and formatted like @twitter')}
```

Comment:
```suggestion
                {errorFor('twitter', 'Should be formatted like @twitter')}
```

Reply:
We should remove the initial state on line 51 or allow sending `"@"` as I strip it on the backend.